packages/pages/src/common/src/template/paths.ts

```diff
@@ -23,12 +21,5 @@ export const getRelativePrefixToRootFromPath = (path: string): string => {
  * paths are supported with import.
  */
 export const convertToPosixPath = (p: string) => {
-  return p.split(winSep).join(path.posix.sep);
-};
-
-/**
- * Convert any path to match the format of the OS. If it already matches no change occurs.
- */
-export const convertToOSPath = (p: string) => {
-  return p.split(path.posix.sep).join(path.sep).split(winSep).join(path.sep);
+  return p.split("\\").join(path.posix.sep);
```

Comment:
Use path.sep in the split

Reply:
The reason for this is that convertToPosixPath would fail on Posix systems. If we are are on on a Posix system and are given a Windows path, convertToPosixPath will not do anything when using path.sep. Using "\\" works on both Windows and Posix.

This case only occurs in unit tests, but I still think we would want this method to work on both.